js/dynamics.js

```diff
@@ -1,19 +1,8 @@
-function onReady(f) {
-  if (document.readyState === 'complete' || (document.readyState !== "loading" && !document.documentElement.doScroll)) {
-    f();
-  }
-  else {
-    document.addEventListener('DOMContentLoaded', f);
-  }
-}
-
 function setLifetime() {
-  const FALCON_EPOCH = 1357365600;
-  const SECONDS_PER_ORBITAL_ROTATION = 31558149.504000004;
```

Comment:
I would guess these were supposed to be humorous... Could we maybe restore the original names?

Reply:
We can keep that, but I would suggest calculating the _seconds per orbital rotations_ dynamically so it is easier to understand what this variable is doing.

Reply:
I think we could keep the previous year length in days of 365,25636.
https://en.wikipedia.org/wiki/Sidereal_year

Reply:
I restored the default value with comment which explains what the variable does.